mmdet/utils/util_distribution.py

```diff
@@ -0,0 +1,59 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import torch
+from mmcv.parallel import MMDataParallel, MMDistributedDataParallel
+
+dp_factory = {
+        'cuda' : MMDataParallel
+    }
+
+ddp_factory = {
+        'cuda' : MMDistributedDataParallel
+    }
+
+def build_dp(model, device = 'cuda', device_ids = 0):
+    """build DataParallel module by device type.
+
+    if device is cuda, return a MMDataParallel model; if device is mlu,
+    return a MLUDataParallel model.
+    """
+    assert device in ['cuda', 'mlu'], "Only available for cuda or mlu devices."
+    if device == 'cuda':
+        model = model.cuda()
+    elif device == 'mlu':
+        from mmcv.device.mlu import MLUDataParallel
+        dp_factory['mlu'] = MLUDataParallel
+        model = model.mlu()
+
+    return dp_factory[device](model, device_ids=device_ids)
+
+
+def build_ddp(model, device = 'cuda', device_ids=0,
+        broadcast_buffers=False, find_unused_parameters=False):
+    """Build DistributedDataParallel module by device type.
```

Comment:
Please use standard docstring like below:
```
"""Build DistributedDataParallel module by device type.

Args:
    model (type): description
    device (str): ...
    device_ids (int): ...
    ...

Returns:
   type: description
```

You may also see example in https://www.sphinx-doc.org/en/master/usage/extensions/example_google.html

Reply:
added